### PR TITLE
Read Me updated

### PR DESCRIPTION
**Pull request by ITHHKN**
- Added screenshot to drag WeScan into Your project 
- Added screenshot to add WeScan into linked frameworks

### CODE REVIEW

**Comment by BasThomas** [comment]
If pull requests should be targeted at `develop`, why did you not choose to make it the default branch? :)

**Comment by ITHHKN** [comment]
Pull Request changed from master branch to dev branch ...

File: README.md [hunk 1]
```diff
@@ -81,9 +81,21 @@ Run `carthage update` to build the framework and drag the built `WeScan.framewor
...
+Simply Add the WeScan FrameWork into project `Embedded Binaries` and `Linked Frameworks and Libraries`.
+
+<p align="center">
+    <img width="900px" src="Assets/LinkedFrameworks.png">
+</p>
+
 ## Usage
 
-1. Make sure that your ViewController conforms to the `ImageScannerControllerDelegate` protocol
+1. Make sure that your ViewController confirms to the `ImageScannerControllerDelegate` protocol
```
Comment (Boris-Em) [inline README.md]: Objects conform to protocols.
For example to quote [this document](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/WorkingwithProtocols/WorkingwithProtocols.html): 
>This chapter describes the syntax to define a formal protocol, and explains how to mark a class interface as conforming to a protocol,

File: README.md [hunk 2]
```diff
@@ -81,9 +81,21 @@ Run `carthage update` to build the framework and drag the built `WeScan.framewor
 
 Just download the project, and drag and drop the "WeScan" folder in your project.
 
+
+<p align="center">
+    <img width="900px" src="Assets/project.png">
+</p>
+
+
+Simply Add the WeScan FrameWork into project `Embedded Binaries` and `Linked Frameworks and Libraries`.
```
Comment (Boris-Em) [inline README.md]: This sentence is not quite correct. Here is a better version:
>  Simply Add the WeScan framework into the project's `Embedded Binaries` and `Linked Frameworks and Libraries`.

Comment (justJS) [reply README.md]: Minor correction: I don't think "Add" should be capitalized in the corrected sentence.

Comment (Boris-Em) [reply README.md]: You're right @justJS!

**Comment by AvdLee** [comment]
@Boris-Em can you decide whether we want to merge this in or not? The PR is quite inactive and we should take an action up on this.

**Comment by AvdLee** [comment]
> If pull requests should be targeted at develop, why did you not choose to make it the default branch? :)

@BasThomas you're totally right. Adjusted this. Thanks!

**Comment by justJS** [comment]
@ITHHKN Could you update the PR as per @Boris-Em's comments? I think @Boris-Em was waiting for that.

**Comment by justJS** [comment]
@boris-em should this be closed as inactive?

**Comment by AvdLee** [comment]
I would wait for a bit more till @ITHHKN improved this. It's a really nice contribution which would be great to have.

File: README.md (hunk 1 above)
Comment (justJS) [reply README.md]: This still hasn't been fixed @ITHHKN

Comment (justJS) [reply README.md]: Also, it seems like you accidentally removed the `### Swift` title

**Comment by Boris-Em** [comment]
Thanks for your contribution @ITHHKN!

**Comment by ITHHKN** [comment]
you are welcome @Boris-Em  .. I will looking forward to work on this project more ..

**Pull request merged**